### PR TITLE
Update caption.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
add paragraph on difference between `<b>`, `<strong>` and the `font-weight` property and when to use each

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/web/html/tables/caption.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/web/html/tables/caption.md?ref=f048a1f2b0fdf67687482748c6a149240ef40b5b)

File: web/html/tables/caption.md [hunk 1]
```diff
@@ -52,7 +55,9 @@ Example:
 </table>
 ```
 
-> 💡 In the example above, the `<strong>` element is used to bring importance to the "Name" and "Starting Date" tags. However, it can be replaced with the `style="font-weight:bold;"` CSS property. 
+> 💡 In the example above, the `<b>` element is used to bring attention to the "Name" and "Starting Date" tags. 
+
+If you'd like to only style the elements bold without bringing attention to them, you should replace it with the `style="font-weight:bold;"` CSS property.
```
Comment (nem035) [inline web/html/tables/caption.md:60]: What is "it" in this case?

Comment (Stefan-Stojanovic) [reply web/html/tables/caption.md:60]: the `<b>` element.

These two are connected: 

> 💡 In the example above, the `<b>` element is used to bring attention to the "Name" and "Starting Date" tags. 

If you'd like to only style the elements bold without bringing attention to them, you should replace it with the `style="font-weight:bold;"` CSS property.



Should I add it as a single note?

Comment (Stefan-Stojanovic) [reply web/html/tables/caption.md]: I've added a span and modified the wording, lmk if it's good

File: web/html/tables/caption.md [hunk 2]
```diff
@@ -55,9 +55,16 @@ Example:
 </table>
 ```
 
-> 💡 In the example above, the `<b>` element is used to bring attention to the "Name" and "Starting Date" tags. 
+In the example above, the `<b>` element is used to bring attention to the "Name" and "Starting Date" tags. 
+
+If you'd like to only style the elements bold without bringing attention to them, you should replace it with a `<span>` element, and apply the `style="font-weight:bold;"` CSS property to it:
```
Comment (nem035) [inline web/html/tables/caption.md:60]:
```suggestion
If you'd like to only style the elements bold without bringing attention to them, you can use a `<span>` element, and apply the `style="font-weight:bold;"` CSS property to it:
```

**Review by nem035 — APPROVED** [review]

**Pull request merged**